### PR TITLE
Jun news

### PR DESCRIPTION
**Pull request by GregSutcliffe**
Newsletter as per usual, let me know all the typos and/or missed entries :)

The author commit enables the use of the "uri" field, which the `jekyll-feed` gem understands and will include in the feed.xml. This is useful because the Discourse integration can read that field and use it to set the owner of the post, given that `author['uri']` is a valid Discourse user.

### CODE REVIEW

File: _posts/2018-06-28-foreman-community-newsletter---june-2018.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+- [smart_proxy_omaha](https://github.com/theforeman/smart_proxy_omaha) updated to 0.0.4
+- [foreman_virt_who_configure](https://github.com/theforeman/foreman_virt_who_configure) updated to 0.2.2
+
+### Acknowledgements
+
+Thanks to everyone who contributes to the Foreman community - patches, plugins,
+testing, bug reports, translations, interviews, presentations, meetups and
+everything else. Your efforts are very much appreciated!
+
+[1_18_rc1]: https://community.theforeman.org/t/1-18-0-rc1-has-been-released/9987
```
Comment (dLobatog) [inline _posts/2018-06-28-foreman-community-newsletter---june-2018.md:187]: Interesting, I didn't know you could do this with markdown (point to these links)

**Comment by GregSutcliffe** [comment]
Helps if you get the date right ...

File: _posts/2018-06-28-foreman-community-newsletter---june-2018.md (hunk 1 above)
Comment (GregSutcliffe) [reply _posts/2018-06-28-foreman-community-newsletter---june-2018.md:187]: yeah, its particularly handy for making a bibliography of common links at the bottom of template posts :)

**Comment by GregSutcliffe** [comment]
Also added the multi-value author code to the post_index, missed that in my first commit

File: _posts/2018-06-28-foreman-community-newsletter---june-2018.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+Staying with the blog, we've also enabled comments on the blog again (after a
+*long* absence), which now uses our
+[forum][https://community.theforeman.org/c/blog] to store the comments. As a
+side effect, you can now read the blog on our forum too, if that's more your
+style. Scroll to the bottom of this post to check it out!
+
+### Redmine upgrade
+
+In a short while, we'll start the long-delayed upgrade to our bug tracker. THis
```
Comment (timogoebel) [inline _posts/2018-06-28-foreman-community-newsletter---june-2018.md]: Typo: `THis` should be `This`

**Comment by GregSutcliffe** [comment]
Merged with a comment about RC3 next week, thanks all!

**Comment by GregSutcliffe** [comment]
Merged as 95ce7e17d

**Pull request closed without merging**